app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  belongs_to :restaurant
+  belongs_to :order
+  validates :name, presence: true,
```

Comment:
Should we consider name to have uniqueness true, so that we don't get multiple user rows with same name ?

Reply:
```suggestion
  validates :name, presence: true, uniqueness: true
```
Same as email addresses, username should be unique